OpenSim/Simulation/Model/ModelVisualPreferences.h

```diff
@@ -66,6 +69,7 @@ class OSIMSIMULATION_API ModelVisualPreferences : public Object {
 private:
     void constructProperties() {
         constructProperty_ModelDisplayHints(ModelDisplayHints());
+        constructProperty_no_visualization(false);
```

Comment:
We should avoid the possibility for double negatives. I would prefer this setting to be named "visualize" or something like that.

Reply:
I don't disagree.

Reply:
I made sure the user facing convenience method name is clear, I definitely welcome better names for the property that will appear in xml and will flip the logic accordingly. I don't particularly like visualize as it is close to "visualizer" so rather overused, but I welcome other opinions.

Reply:
The description is "Flag to indicate whether or not to show geometry from mesh files, default to false." so I assume analytic geometry would still visualize. If that's the case, then perhaps put "Mesh" in the name of the property, like `loadMeshFiles`?

Reply:
@tkuchida I would have called it show_meshes but I wanted to get feedback on (item 3) whether we want to use this more widely to avoid calling generateDecorations altogether which would affect all geometry but may be more efficient.

Reply:
This seems somewhat confusing to me particularly in context of exiting `Model::get/setUseVisualizer()`. I realize that this is for the API visualizer and not the GUI visualizer. If you are not using the API visualizer (and you are not in the GUI) you would expect zero overhead/errors related to visualization.  If you are in the GUI or have the visualizer on, then visualize should be true. You shouldn't have to turn two flags on.

I like the idea of bypassing `generateDecorations()` altogether if there is no visualization.

Can we simplify the interface and only have one internal `visualize` flag without exposing any properties?  If `getUseVisualizer` is true then it is true. The GUI turns it to true but otherwise it is false.